filetests/isa/x86/simd-comparison-run.clif

```diff
@@ -217,3 +217,17 @@ ebb0:
     return v8
 }
 ; run
+
+function %fcmp_lt_nans_f32x4() -> b1 {
+ebb0:
+    v0 = vconst.f32x4 [NaN NaN NaN NaN]
+    v1 = vconst.f32x4 [NaN NaN NaN NaN]
```

Comment:
I'm a bit concerned that this test isn't broad enough.  And that having the same operand-pair in all 4 lanes might be a missed opportunity.  Would it be possible to use it to compare not only a NaN-pair, but also a NaN vs a non-NaN, and the other way round?  Get more coverage from just this one test, I mean.  I looked through the rest of the file but wasn't sure that I sure NaN vs non-NaN tests for fcmp_lt anywhere else.  (I may have missed them).

Reply:
That's a good point; let me expand that test.